src/ui.form-calendar/configs/default.config.js

```diff
@@ -32,11 +32,11 @@ export default /*tw*/ {
   currentDay: "hover:bg-brand-100 border-2 border-brand-900",
   anotherMonthDay: "hover:bg-brand-100 text-brand-400",
   monthView: "grid grid-cols-4 items-center justify-center",
-  month: "mx-auto flex h-12 w-full rounded-lg hover:!bg-brand-900 hover:text-white",
+  month: "{UButton} mx-auto flex h-12 w-full rounded-lg hover:!bg-brand-900 hover:text-white",
   selectedMonth: "bg-brand-900 hover:bg-brand-900 text-white",
   activeMonth: "bg-brand-100",
   yearView: "grid grid-cols-4 items-center justify-center",
-  year: "mx-auto flex h-12 w-full rounded-lg hover:!bg-brand-900 hover:text-white",
+  year: "{UButton} mx-auto flex h-12 w-full rounded-lg hover:!bg-brand-900 hover:text-white",
```

Comment:
Calendar component have much more nested Vueless components. Please check `.vue` files and fix it.